src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -728,15 +728,14 @@ def load_arguments(self, _):
                    help='Specify it to create new virtual machines to upgrade the scale set, rather than updating the existing virtual machines.')
 
     with self.argument_context('vmss create', arg_group='Network Balancer') as c:
-        LoadBalancerSkuName = self.get_models('LoadBalancerSkuName', resource_type=ResourceType.MGMT_NETWORK)
         c.argument('application_gateway', help='Name to use when creating a new application gateway (default) or referencing an existing one. Can also reference an existing application gateway by ID or specify "" for none.', options_list=['--app-gateway'])
         c.argument('app_gateway_capacity', help='The number of instances to use when creating a new application gateway.')
         c.argument('app_gateway_sku', help='SKU when creating a new application gateway.')
         c.argument('app_gateway_subnet_address_prefix', help='The subnet IP address prefix to use when creating a new application gateway in CIDR format.')
         c.argument('backend_pool_name', help='Name to use for the backend pool when creating a new load balancer or application gateway.')
         c.argument('backend_port', help='When creating a new load balancer, backend port to open with NAT rules (Defaults to 22 on Linux and 3389 on Windows). When creating an application gateway, the backend port to use for the backend HTTP settings.', type=int)
         c.argument('load_balancer', help='Name to use when creating a new load balancer (default) or referencing an existing one. Can also reference an existing load balancer by ID or specify "" for none.', options_list=['--load-balancer', '--lb'])
-        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', options_list=['--lb-sku'], arg_type=get_enum_type(LoadBalancerSkuName),
+        c.argument('load_balancer_sku', resource_type=ResourceType.MGMT_NETWORK, min_api='2017-08-01', options_list=['--lb-sku'], arg_type=get_enum_type(['Basic', 'Standard', 'Gateway']),
```

Comment:
`LoadBalancerSkuName` has two values(`Basic`, `Standard`) before version `2021-02-01` and three values(`Basic`, `Standard`, `Gateway`) after.

Reply:
This change comes from your PR.